### PR TITLE
Feature/SOF-7020

### PR DESCRIPTION
**Pull request by VsevolodX**

### CODE REVIEW

File: materials/Graphene.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
+{
```
Comment (timurbazhirov) [inline materials/Graphene.json:1]: Let's add sources per https://github.com/Exabyte-io/esse/blob/7271583763bcfc9d7f61130aaccef16681eabc66/schema/core/reference/literature.json

Comment (timurbazhirov) [reply materials/Graphene.json:1]: We are using 4 spaces as tabs everywhere, let's add JSON linter to github workflow

File: materials/WS2.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
+{
+  "lattice": {
+    "a": 3.190731,
+    "b": 3.19073,
+    "c": 14.2024,
```
Comment (timurbazhirov) [inline materials/WS2.json:5]: Same as above

File: materials/Graphene.json (hunk 1 above)
Comment (VsevolodX) [reply materials/Graphene.json:1]: Ok. I've linted it. Although highly disagree on the alphabetical sorting of keys. It makes harder to work with the entities for humans.

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**